snapcraft/plugins/v1/python.py

```diff
@@ -124,6 +125,7 @@ def schema(cls):
             "default": "python3",
             "enum": ["python2", "python3"],
         }
+        schema["properties"]["prebuilt-wheel-dir"] = {"type": "string", "default": ""}
```

Comment:
Remove default of empty string:
```suggestion
        schema["properties"]["prebuilt-wheel-dir"] = {"type": "string"}
```